godot-macros/src/util.rs

```diff
@@ -431,22 +427,40 @@ mod tests {
     use proc_macro2::TokenStream;
     use quote::quote;
 
-    macro_rules! hash_map {
+    macro_rules! kv_map {
         (
             $($key:expr => $value:expr),*
             $(,)?
         ) => {
             {
                 let mut map = std::collections::HashMap::new();
                 $(
-                    map.insert($key, $value);
+                    map.insert(ident($key), $value);
```

Comment:
Nice change! 👍

Reply:
We can do even better: since the keys are always identifiers, we don't even need the quotes.